src/k8s-engine/wsl.ts

```diff
@@ -307,6 +308,39 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
         this.emit('service-changed', services);
       });
       this.activeVersion = desiredVersion;
+
+      // Temporary workaround: ensure root is mounted as shared -- this will be done later
+      // Right now the builder pod needs to be restarted after the remount
+      try {
+        await childProcess.exec('wsl --user root -d k3s mount --make-shared /');
+        console.log('Waiting for ensuring root is shared');
+        await util.promisify(setTimeout)(60_000);
```

Comment:
For the future: check /proc/self/mountinfo` for `shared:` in the 7th field.
But this is fine for now.

Reply:
This is all a temporary workaround until `run-k3s` does the mounting, and all this code will be dropped.